src/routes/relay/relay.controller.spec.ts

```diff
@@ -151,7 +151,7 @@ describe('RelayController', () => {
 
       describe('execTransaction', () => {
         it('should return a 201 when the body is a valid execTransaction call', async () => {
-          const data = await getMockExecTransactionCalldata({
+          const data = getMockExecTransactionCalldata({
```

Comment:
My IDE found a lot of unnecessary `await`s and quickfixed them.

Reply:
Suggestion: I'd leave these changes to a separate PR (or non-squashed commit). Some reasons for it:
1. Makes the PR easier to review as it tackles one specific issue
2. Makes reverts easier to manage. If we want to revert some functionality in the future (eg.: TypeChain) then we are also reverting the `await`s.
3. We can approve the `await` change while we discuss the `TypeChain` changes so one change is not "blocked" by the other.

This change is simple enough that in my opinion 1 doesn't need to be considered. 2 and 3 is more relevant for this PR in particular.

Reply:
All valid points. I've removed the changes from this PR and created [a new one](https://github.com/safe-global/safe-gelato-relay-service/pull/87) with them instead.